bigtable-dataflow-parent/bigtable-hbase-beam/src/test/java/com/google/cloud/bigtable/beam/CloudBigtableIOIntegrationTest.java

```diff
@@ -27,6 +26,7 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
+import java.util.logging.Logger;
```

Comment:
I think this requires a bit of thought. We are really inconsistent with the loggers that are used in the project. Lets chat offline